app/__init__.py

```diff
@@ -174,3 +209,20 @@ def __init__(self, message, invalid_pages=None, page_count=None, code=400):
         self.invalid_pages = invalid_pages
         self.code = code
         self.page_count = page_count
+
+
+class QueueNames:
+    LETTERS = 'letter-tasks'
+    TEMPLATE_PREVIEW = 'template-preview-tasks'
+
+
+class TaskNames:
+    PROCESS_SANITISED_LETTER = 'process-sanitised-letter'
+
+
+queue_prefix = {
+    'development': os.environ.get('NOTIFICATION_QUEUE_PREFIX', 'development'),
```

Comment:
it'd be nice if it throw an exception if your environment is `development` and your NOTIFICATION_QUEUE_PREFIX is unset. Might need this to be made in to a `get_queue_prefix(environment)` function though

Reply:
I've made a function to check (rebased into the first commit). The downside is that you now need to have `NOTIFICATION_QUEUE_PREFIX` set for both apps, even though the Flask app doesn't use it. But I guess that's probably ok

Reply:
On second thoughts, having tried this a bit more I'm not sure that it is an improvement to throw an exception when `NOTIFICATION_QUEUE_PREFIX` is unset. The problem is that when you run the Flask app it raises the error, but the message about queue prefix not being set is pretty far up the stack trace and at first glance it looks like the issue is a Gunicorn one. So it's quite hard to tell what the problem is when you're running the Flask app

Might be better to undo this change and just default to a queue prefix of `development` like we do in api and antivirus

Reply:
Changed here https://github.com/alphagov/notifications-template-preview/pull/399/commits/9bc308ee1adf4cf2c7cf76dfe3b9bd1ebcd1a316 to the approach we talked about of having the Makefile check that the environment variable is set. `make run-with-docker` (to run the Flask app) doesn't give any errors if `NOTIFICATION_QUEUE_PREFIX` is unset, so we don't need to add a default value.